README.md

```diff
@@ -11,7 +11,7 @@ Follow the videos in this and next lessons to implement the page block by block
 - Start writing styles in the `main.scss` file. It is be explained in the `Sass` lesson
 - **DON'T** try to do it `Pixel Perfect` - implement it the most `simple` way so it looks similar;
 - When done check yourself using the [CHECKLIST](./checklist.md) when finished;
-- Deploy and create a Pull Request with a [DEMO LINK](https://<your_account>.github.io/layout_landing-page/)
+- Deploy and create a Pull Request with a [DEMO LINK](https://unholysearcher.github.io/layout_landing-page/)
```

Comment:
Add this link to PR description
![image](https://github.com/mate-academy/layout_landing-page/assets/70009205/de2e838c-7aed-49d5-a20a-53648142dcef)